internal/variants/variants.go

```diff
@@ -82,16 +113,32 @@ func (q *Query) whereClause() string {
 	add := func(format string, args ...interface{}) {
 		clauses = append(clauses, fmt.Sprintf(format, args...))
 	}
-	simpleClause := func(dbColumn, value string) {
+	stringClause := func(dbColumn, value string) {
 		if dbColumn != "" && value != "" {
```

Comment:
You could instead do:

simpleClause := func(column string, value interface{}) {
  switch value := value.(type) {
    case string:
      add("%s='%s'", value)
    case *int64:
      add("%s=%d", *value)
  }
}

Why do you need the column name check?

Reply:
done, I don't need to check column name, I removed that.